packages/next-templates/src/app/productpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,164 @@
+'use client';
+
+import {
+  Article,
+  BreadcrumbNav,
+  BreadcrumbNavLink,
+  BreadcrumbNavSeparator,
+  ButtonLink,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { PreFooterNav } from '@/components/PreFooterNav/PreFooterNav';
+import { IconChevronRight } from '@tabler/icons-react';
+
+import '@utrecht/design-tokens/dist/index.css';
+import '@/app/styling/css/productpagina-klachtenformulier.css';
+import '@/app/styling/css/detail-page.css';
+
+import { useEffect } from 'react';
+
+export default function home() {
+  const pageTitle = 'Klacht over de gemeente doorgeven';
+  const websiteLabel = 'gemeente Voorbeeld';
+
+  useEffect(() => {
+    if (typeof document?.title === 'string') {
+      document.title = `${pageTitle} - ${websiteLabel}`;
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+      <BreadcrumbNav label="Kruimelpad" className="breadcrumb-nav">
```

Comment:
de breadcrumb uitlijning is staat fout als je naar de graffiti melding pagina gaat voor dat je de formulier ziet dan zie je hoe de breadcrumb uitgelijnd is

Reply:
Goed punt! Dit komt (denk ik) omdat ik de breadcrumb buiten de main heb geplaatst. Dit had ik als richtlijn gevonden op NLDS. Op de graffiti pagina staat hij nog in main (heb ik een issue voor aangemaakt). 

Ik wist ook niet zeker of ik de graffiti pagina als voorbeeld kon aannemen, aangezien iedere pagina op gv (behalve wmebv) een beetje improv is. 🤓 In het Figma ontwerp staat de breadcrumb in ieder geval helemaal links bovenin, dat heb ik voor de zekerheid gewoon aangehouden.

<img width="500" alt="Scherm­afbeelding 2024-04-08 om 14 10 53" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/e4da58e9-627c-4fa6-b413-69800f13e13a">

De wijzigingen van Ryan (in globals.css) gaan ook nog iets met de layout van deze pagina doen. Geen idee wat dat met deze pagina gaat doen, die PR moet nog gemerged worden namelijk.

Wat is wijsheid? (optie 1 of 2 lijkt mij het meest waarschijnlijk)

1. Zal ik het gewoon zo laten, dat deze pagina het voorbeeld is voor als de breadcrumb issue opgepakt wordt?
2. Even wachten tot Ryan's PR gemerged is en dan mijn breadcrumb aanpassen naar hoe het op de graffiti pagina is?
3. Mijn breadcrumb nu aanpassen naar de breadcrumb zoals hij op de graffiti pagina staat?
4. Iets anders?

Reply:
ik denk dat ryan's pr snel gemerged moet worden maar dat is main content spacing en de breadcrumb die van jouw is buiten de content. 
Ook denk ik dat de spacing van de breadcrumb uitgelijned kan zijn met de navigation maar dat moeten we aan @Robbert vragen.

Reply:
Ik vind het okay als de breadcrumb eventjes lelijk is en we het gauw fixen, vind het wel belangrijk dat 'ie buiten `main` staat.

Reply:
Nice, dan laat ik de breadcrumb op deze pagina voor nu zoals hij is en sluit ik deze convo. 

Het fixen van de breadcrumb zal ik als subtask aan de breadcrumb issue toevoegen die ik eerder had aangemaakt. Die gaat namelijk over verschillende pagina's GV breed.